application/world/services/auto_bible_generator.py

```
@@ -17,6 +17,41 @@
+# ============================================================================
+# JSON 输出稳定性增强 - Prompt 常量
+# ============================================================================
+USER_PROMPT_SUFFIX = """
+
+请按照以下json格式进行输出，可以被Python json.loads函数解析。只给出JSON，不作解释，不作答：
+```json
+"""
+
+
+def parse_json_from_response(rsp: str):
+    """从LLM响应中解析JSON，支持```json包裹格式"""
```

Ruff (0.15.10)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)